src/packages.builds

```diff
@@ -8,14 +8,33 @@
     <BuildAllOSGroups Condition="'$(FilterToOSGroup)' != ''">false</BuildAllOSGroups>
   </PropertyGroup>
 
+  <!--
+    the following packages are built and shipped from TFS, so while we still want
+    to build them we don't want to ship them.  by setting NonShippingPackage to true
+    the packages will still be built but placed in a different location.
+  -->
   <ItemGroup>
-    <Project Include="Native\pkg\**\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
+    <NativeNonShippingPackage Include="Native\pkg\runtime.native.System.Data.SqlClient.sni\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'" />
+    <NativeNonShippingPackage Include="Native\pkg\runtime.native.System.IO.Compression\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'" />
+    <NonShippingPackage Include="System.Data.SqlClient\pkg\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'" />
+    <NonShippingPackage Include="System.IO.Compression\pkg\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'" />
```

Comment:
Is the `Condition="'$(SkipManagedPackageBuild)' != 'true'"` necessary up here? As far as I can tell, the condition is fine left until later when these items might be used.